feature/dashboard/src/main/java/com/simprints/feature/dashboard/settings/syncinfo/SyncInfoViewModel.kt

```diff
@@ -146,6 +149,12 @@ internal class SyncInfoViewModel @Inject constructor(
     }
 
     private fun load() = viewModelScope.launch {
+        // if device is not safe stop all sync operations
+        // users can root the device after login and we need to stop sync operations
+        if (isDeviceSafe().not()) {
+            return@launch
```

Comment:
Should we not add a specific message on the dashboard when this is the case, as it would be easier to debug if it happens in a project?

Reply:
@ybourgery, which dashboard? 
if you mean the SID dashboard, I don't think we should notify the user that the sync operations are disabled becasue we is rooted the device.

Reply:
Yes, the SID dashboard. If we don't notify it and a user is complaining because their device is not syncing, I'm not sure we could find that the root cause is because the device is rooted